packages/data-table-server/examples.http

```diff
@@ -71,6 +71,44 @@ Authorization: {{authorization}}
   ]
 }
 
+### Fetch data from tupaia_data_elements_metadata data-table
+POST http://{{host}}/{{version}}/dataTable/tupaia_data_elements_metadata/fetchData HTTP/1.1
+content-type: {{contentType}}
+Authorization: {{authorization}}
+
+{
+    "dataElementCodes": ["PSSS_AFR_Cases"]
+}
+
+
+### Fetch data from tupaia_data_group_metadata data-table
+POST http://{{host}}/{{version}}/dataTable/tupaia_data_group_metadata/fetchData HTTP/1.1
+content-type: {{contentType}}
+Authorization: {{authorization}}
+
+{
+    "dataGroupCode": "SPF"
+}
+
+
+### Fetch data from dhis_data_elements_metadata data-table
+POST http://{{host}}/{{version}}/dataTable/dhis_data_elements_metadata/fetchData HTTP/1.1
+content-type: {{contentType}}
+Authorization: {{authorization}}
+
```

Comment:
I am actually on the fence between `data_elements_metadata` and `data_element_metadata` without the `s`...